backend/rates/gecko.go

```diff
@@ -100,6 +100,7 @@ var (
 		"SEK": "sek",
 		"PLN": "pln",
 		"CZK": "czk",
+		"sat": "BTC",
```

Comment:
add comment that this is handled/converted manually for clarity

Reply:
Added.